examples/md-flexible/src/parsing/CLIParser.cpp

```diff
@@ -569,3 +575,67 @@ void CLIParser::inputFilesPresent(int argc, char **argv, MDFlexConfig &config) {
 
   opterr = opterrBefore;
 }
+
+void MDFlexParser::CLIParser::printHelpMessage(
+    std::ostream &ostream, const std::string &relPathOfExecutable,
+    const std::vector<MDFlexConfig::MDFlexOptionInterface> &relevantOptions) {
+  ostream << "Usage: " << relPathOfExecutable << "\n";
```

Comment:
comment, how you can maintain this to be zsh completion conform

Reply:
We do not derive the completions from `--help` anymore but generate a completion file directly. See `createZSHCompletionFile()`.